applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -24,6 +24,14 @@ LOG_MODULE_REGISTER(cis_gateway, CONFIG_BLE_LOG_LEVEL);
 
 #define HCI_ISO_BUF_ALLOC_PER_CHAN 2
 #define CIS_CONN_RETRY_TIMES 5
+/* Scan interval as two times of ISO interval = 20 ms (0x20 * 0.625) */
+#define BLE_ACL_SCAN_INTERVAL 0x0020
```

Comment:
This must be calculated using the ISO interval

Reply:
Update: using API to fetch the current codec ISO interval setting.